src/common/FileSystem.cpp

```diff
@@ -909,6 +909,20 @@ class ZipArchive {
 		return fileInfo.uncompressed_size;
 	}
 
+	// Check if currently open file is a symbolic link
+	bool CheckSymlink(std::error_code& err) const
```

Comment:
Rename function to IsSymlink() to be more clear. CheckSymlink sounds like its checking for validity.

Reply:
I agree